### ISSUE TITLE
Improve matching of canvas skewb orientation and scramble orientation

### ISSUE THREAD

**Issue by AnnikaStein** [AnnikaStein/SkewbPage#10]
✅ FLT:
- All perfect, use this as reference!

✅ L2L:
- Skewb looks correct (matching scramble),
- but is not oriented with white up green left (as in WCA notation, therefore does not necessarily match what user sees).

✅ OLT:
- Remove rotations at the beginning and end from what is visible to the user.
- Skewb does match what the user sees,
- target is also to get white on the top and green on the left.

In summary:
- [x] No rotations in scramble
- [x] Image = What user sees
- [x] Image = User = White top, green left

**Comment by AnnikaStein** [comment]
Partially done in https://github.com/AnnikaStein/SkewbPage/commit/ec26329f9800b2222eaf1b1794726718eb798e56
